### DISCUSSION TITLE
Sport TV/ABola/Caça e Pesca/Caçavision Channels?

### DISCUSSION THREAD

**Discussion by JBar1017** [LITUATUI/M3UPT#87] [General]
Hey there! Great work on this. Great to see some great Portuguese Channels and in great quality. Will there be any Sport TV Channels, ABola channel, Caça e Pesca Channel, and Caçavision channel coming up? Again, keep this coming so amazing this all is. You've made my family very happy with this! Big soccer fan here too! 🇵🇹☺️

**Comment by LITUATUI** [comment]
This list is public, therefore only contains public streams.

**Comment by JBar1017** [comment]
Thanks for the clarification 😃
